lib/aac/index.js

```diff
@@ -110,6 +129,13 @@ AacStream = function() {
       }
       byteIndex++;
     }
+    bytesLeft = everything.length - byteIndex;
+
+    if (bytesLeft > 0) {
+      everything = everything.subarray(byteIndex);
+    } else {
+      everything = null;
```

Comment:
Not important but I'd lean towards making `everything` an empty typed array when there's no data to process and removing the need to check for nulls.